pkg/infrastructure/gcp/clusterapi/ignition.go

```diff
@@ -35,7 +41,8 @@ const (
 // EditIgnition attempts to edit the contents of the bootstrap ignition when the user has selected
 // a custom DNS configuration. Find the public and private load balancer addresses and fill in the
 // infrastructure file within the ignition struct.
-func EditIgnition(ctx context.Context, in clusterapi.IgnitionInput) ([]byte, error) {
+func EditIgnition(ctx context.Context, in clusterapi.IgnitionInput) ([]byte, []byte, error) {
```

Comment:
Is there any scenario the installer has where the bootstrap AND master don't need to both be edited? Wondering If this needs to be separated into two functions for EditBootstrapIgnition and EditControlPlaneIgnition.

Reply:
The control-plane pointer ignition is updated in its own method updatePointerIgnition(). 
I had considered the scenario that you mentioned. I am sure there might be times when only one Ignition needs to be updated. But, I am unable to come up with an example now.

These methods to manipulate the ignition are very specific to the exact changes needed for custom-dns. I think it is OK to design these just for this purpose without worrying about other scenarios where we might need Ignition editing functions that are completely generic and independent of each other.

Reply:
I am ok with this. I just thought we should consider it.